scan/src/components/oracle-script/OracleScriptExecute.re

```diff
@@ -152,136 +118,44 @@ let resultRender = result => {
-                            code=true
-                            color=Colors.gray8
-                            weight=Text.Bold
-                          />
-                        </div>
-                      </div>
-                    )
-                    |> Belt_List.toArray
-                    |> React.array}
-                 </div>
-               </div>
-               <VSpacing size=Spacing.lg />
-               <div className={Styles.hFlex(`auto)}>
-                 <HSpacing size=Spacing.lg />
-                 <div className={Styles.vFlex(`px(220), `auto)}>
-                   <Text
-                     value="PROOF OF VALIDITY"
-                     size=Text.Sm
-                     color=Colors.gray6
-                     weight=Text.Semibold
-                     height={Text.Px(15)}
-                   />
-                 </div>
-                 <div className={Styles.vFlex(`px(660), `auto)}>
-                   <Text
-                     value={proof |> JsBuffer.toHex}
-                     height={Text.Px(15)}
-                     code=true
-                     ellipsis=true
-                   />
-                 </div>
-               </div>
-               <VSpacing size=Spacing.md />
-               <div className={Styles.hFlex(`auto)}>
-                 <HSpacing size=Spacing.lg />
-                 <div className={Styles.vFlex(`px(220), `auto)} />
-                 {copyButton(~data=proof)}
-                 <HSpacing size=Spacing.lg />
-                 {extLinkButton()}
-               </div>
-             </>
-           : <div className={Styles.hFlex(`auto)}>
-               <HSpacing size=Spacing.lg />
-               <div className={Styles.resultWrapper(`px(220), `px(12), `zero, `auto)}>
-                 <Text
-                   value="WAITING FOR OUTPUT AND PROOF"
-                   size=Text.Sm
-                   color=Colors.gray6
-                   weight=Text.Semibold
-                 />
-               </div>
-               <div className={Styles.resultWrapper(`px(660), `px(40), `zero, `auto)}>
-                 <ProgressBar reportedValidators=3 minimumValidators=4 requestValidators=5 />
-               </div>
-             </div>}
-      </div>
-    </>;
+  | Success(txResponse, schema) => <OracleScriptExecuteResponse txResponse schema />
   };
 };
 
 [@react.component]
-let make = (~code: Hash.t) => {
-  let params = ["Symbol", "Multiplier"]; // TODO, replace this mock by the real deal
-  let numParams = params->Belt_List.length;
-  // TODO: wire up later
-  Js.Console.log(code);
-
-  // let (callDataList, setCallDataList) = React.useState(_ => Belt_List.make(numParams, ""));
-
-  // let (result, setResult) = React.useState(_ => Nothing);
+let make = (~id: ID.OracleScript.t, ~schemaOpt: option(string)) => {
+  let (_, dispatch) = React.useContext(AccountContext.context);
+
+  let schema = schemaOpt->Belt_Option.getWithDefault("");
```

Comment:
If there's no schema, we should just show to use that schema is not available. Doing default like this is error prone.